accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -27,18 +27,39 @@
 
 import java.util.List;
 
+/**
+ * Service that provides various accessioning features for objects
+ *
+ * @param <MODEL> Specifies the type of the Accession model
+ * @param <HASH> Hash key used for accessioning
+ * @param <ACCESSION> Accession ID of the object
```

Comment:
Using 'accession' and 'ID' together is redundant:

```suggestion
 * @param <ACCESSION> Type of the accessions that identify a particular model
```

Applies to other methods.